code/datums/gamemode/factions/bloodcult/bloodcult_runespells.dm

```diff
@@ -741,6 +741,10 @@
 	flick("rune_convert_start",conversion)
 	playsound(R, 'sound/effects/convert_start.ogg', 75, 0, -4)
 
+	var/obj/item/device/gps/secure/SPS = locate() in victim
```

Comment:
This won't trigger if you wear your SPS on a belt or in a packback. Intended ?

Reply:
Do they trigger on death if they're in a belt of a pack?